src/simplewallet/simplewallet.cpp

```diff
@@ -2320,17 +2320,8 @@ bool simple_wallet::set_ask_password(const std::vector<std::string> &args/* = st
 bool simple_wallet::set_unit(const std::vector<std::string> &args/* = std::vector<std::string>()*/)
```

Comment:
This can go further and just rip out this whole `set_unit` call + the simplewallet code and command that wires up to it.